src/vue/backend/src/App.vue

```diff
@@ -1,12 +1,24 @@
 <template>
   <idx-block className="app">
-    <idx-block className="nav">
-      <router-link to="/">Home</router-link> |
-      <router-link to="/about">About</router-link>
-    </idx-block>
-    <router-view/>
+    <Navbar></Navbar>
+    <page>
+        <transition name="router" mode="out-in">
```

Comment:
I believe the name will need to be prefixed when you get transition styles on this